src/receipt/ReceiptPage.jsx

```diff
@@ -0,0 +1,463 @@
+import {
+  ActionRow, Alert, Badge, DataTable, Hyperlink, PageBanner,
+} from '@edx/paragon';
+import { Info } from '@edx/paragon/icons';
+
+import messages from './ReceiptPage.messages';
+
+// Actions
+import { fetchOrder } from './actions';
+import { receiptSelector } from './selectors';
+import { PageLoading } from '../common';
+
+const { LMS_BASE_URL } = getConfig();
+const DASHBOARD_URL = `${LMS_BASE_URL}/dashboard`;
+const FIND_COURSES_URL = `${LMS_BASE_URL}/courses`;
+class ReceiptPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: true,
+    };
+  }
+
+  componentDidMount() {
+    const orderNumber = this.props.history.location.search.split('=')[1];
+    this.props.fetchOrder(orderNumber);
+  }
+
+  getConfirmMessage() {
+    return this.props.order.lines.map(({
+      product,
+    }) => (
+      product.is_enrollment_code_product ? (
+        <span key={product.id}>
+          {this.props.intl.formatMessage(messages['ecommerce.receipt.confirm.message.enrollment.code'])}
+          <Hyperlink data-hj-suppress destination="mailto:{order.user.email}">{this.props.order.user.email}</Hyperlink>
+        </span>
+      ) : (
+        <span key={product.id}>
+          {this.props.intl.formatMessage(messages['ecommerce.receipt.confirm.message'])}
+        </span>
+      )
+    ));
+  }
+
+  getProductTrackingVariable() {
+    const productTracking = this.props.order.product_tracking;
+    return `var awinProductTrackingData = ${productTracking}`;
+  }
+
+  renderDiscountByType(discounts) {
+    return discounts.map(discount => {
+      const discountAmount = (
+        <FormattedNumber
+          value={-Math.abs(discount.amount)}
+          style="currency" // eslint-disable-line react/style-prop-object
+          currency={discount.currency}
+        />
+      );
+      if (discount.code) {
+        return (
+          <>
+            <span data-hj-suppress>{discount.code}</span>
+            <ActionRow.Spacer />
+            <span>{discount.benefit_value}% off</span>
+            <ActionRow.Spacer />
+            <span>{discountAmount}</span>
+          </>
+        );
+      }
+      if (discount.contains_offer) {
+        if (discount.enterprise_customer_name) {
+          return (
+            <>
+              <FormattedMessage
+                id="ecommerce.receipt.table.order.discount.message.enterprise"
+                defaultMessage="Discount of type {offerType} provided by {enterpriseCustomer}"
+                key={discount.id}
+                values={{
+                  offerType: discount.offer_type,
+                  enterpriseCustomer: discount.enterprise_customer_name,
+                }}
+              />
+              <ActionRow.Spacer />
+              <span>{discountAmount}</span>
+            </>
+          );
+        // eslint-disable-next-line no-else-return
+        } else if (discount.condition_name === 'dynamic_discount_condition') {
```

Comment:
I intentionally disabled it because I thought for readability it was better with the `else` and not just ending in the `else if`, I don't always understand ESLint logic tbh, but will update here